packages/block-library/src/post-title/block.json

```diff
@@ -12,6 +12,19 @@
 		"level": {
 			"type": "number",
 			"default": 2
+		},
+		"makeLink": {
```

Comment:
should it be more "isLink"?

Reply:
hm.. it seems better yes. Made a follow up here: https://github.com/WordPress/gutenberg/pull/25397